web/models/epjs_user.ex

```diff
@@ -0,0 +1,23 @@
+defmodule Healthlocker.EPJSUser do
+  use Healthlocker.Web, :model
+
+  schema "epjs_users" do
+    field :patient_id, :integer
```

Comment:
alright cool, I'll change it to the same capitalisation. What do you mean use a string here? Can't we still do an atom like `:Patient_ID`? or do you mean the type?